lib/pure/httpclient.nim

```diff
@@ -121,12 +121,59 @@
 ## if however data does not reach the client within the specified timeout a
 ## ``TimeoutError`` exception will be raised.
 ##
+## Examples:
```

Comment:
Please write instead "Here is how to set a timeout when creating a \`\`HttpClient\`\` instance". Same for below. "Examples" is too vague.